Formula/goctl.rb

```diff
@@ -0,0 +1,19 @@
+class Goctl < Formula
+  desc "Generates server-side and client-side code for web and RPC services"
+  homepage "https://go-zero.dev"
+  url "https://github.com/zeromicro/go-zero/archive/refs/tags/tools/goctl/v1.3.5.tar.gz"
+  sha256 "cbb131ba03fd26e4871ce9543cf8aa64fa1547d6db5c10390f512e72d095a722"
+  license "MIT"
+
+  depends_on "go" => :build
+
+  def install
+    chdir "tools/goctl" do
+      system "go", "build", *std_go_args(ldflags: "-s -w", output: bin/"goctl"), "goctl.go"
```

Comment:
This is the default 
```suggestion
      system "go", "build", *std_go_args(ldflags: "-s -w"), "goctl.go"
```

Reply:
ok thanks

Reply:
So it shouldn't be specified, because it's the same as the default